### DISCUSSION TITLE
could this  awesome package do the following?

### DISCUSSION THREAD

**Discussion by khaledmurtaja** [gdelataillade/alarm#252] [Q&A] [answered]
So this is my use case, a client came for me with a beautiful figma design for an alarm app, as i saw from the design a wakeup screen should appear when alarm rings, it may contain math problems to solve to make sure the user is awake which is a brilliant idea, this screen must appear even if the app is killed or in the background or over any other apps , in both android & IOS so my question is this applicable using the package? ( i am not talking about the UI part [tell me if you have any notes] , i am talking about the functionality in general, so i do not need small pop up or notification i need a full screen which may contain any other actions like ads or math problems

**Comment by gdelataillade** [comment, marked as answer]
Hi @khaledmurtaja,

Thanks for your interest in the package. On iOS, it is not possible to show a screen if app is in the background or killed. Besides, alarm won't ring if app was killed on iOS.

Let me know if you have any questions.
